ccmlib/common.py

```diff
@@ -458,10 +460,43 @@ def scylla_extract_install_dir_and_mode(install_dir):
     return install_dir, scylla_mode
 
 
+def check_file_exists(path_to_file):
+    if os.path.exists(path_to_file):
+        return True
+    return False
+
+
+def _wait_until_false(wait_seconds, func, kwargs):
+    start_time = time.time()
+    wait_time_sec = 5
+    while True:
+        if not func(**kwargs):
+            return True
+        elapsed = time.time() - start_time
+        if elapsed >= wait_seconds:
+            return False
+        time.sleep(wait_time_sec)
+        if elapsed + wait_time_sec > wait_seconds:
+            wait_time_sec = wait_seconds - elapsed
+        elif wait_time_sec <= 16:
+            wait_time_sec *= 2
+
+
+def wait_for_parallel_download_finish(placeholder_file):
+    if check_file_exists(placeholder_file):
+        if not _wait_until_false(3600, check_file_exists, kwargs={'path_to_file': placeholder_file}):
```

Comment:
call `_wait_until_false` with keyword arguments

Reply:
done

Reply:
In Python we can call function/method using
- positional arguments 
- keyword arguments

Please use kw arguments when calling function 

```_wait_until_false(wait_seconds=3600, func=check_file_exists...)```